app/views/audits/show.html.erb

```diff
@@ -51,6 +51,10 @@
         <% end %>
         </tbody>
       </table>
+      <% if @audit.comment %>
+        <%= alert(:class => 'alert-info', :header => '',
```

Comment:
two concerns:
1. its inside the table, yet its not related to any of the table actions, probably needs a re-review of the entire show diff part, /cc @Rohoover 
2. the alert will be shown to any other audit that has a comment, not just key pairs. e.g. templates.